app/src/main/java/com/eis/phoneringer/MainActivity.java

```diff
@@ -4,11 +4,14 @@
 
 import android.os.Bundle;
 
+
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
```

Comment:
All you did to this file was... Adding spaces? You should revert these changes in order to have a cleaner pull request

Reply:
Not really, I cleared my MainActivity and reformatted it, but it looks like th MainActivity in the main repository is NOT formatted...